### ISSUE TITLE
ReactSvgPanZoomLoader doesn't support dynamically built SVG content as XML

### ISSUE THREAD

**Issue by isaias-freitas-voith** [nufaylr/react-svg-pan-zoom-loader#33]
Currently, the ReactSvgPanZoomLoader only supports loading of an SVG file via prop "src". Since it uses SvgLoader under the hood, why not also accept an SVG xml content like SvgLoader does via prop "svgXML"? 
e.g.
```
<SvgLoader svgXML={props.svgXML}>
    {props.proxy}
</SvgLoader>
```

**Comment by acenturyandabit** [comment]
See #38
Before it is merged, you can run `npm install acenturyandabit/react-svg-pan-zoom-loader`. You may need your own type declaration file .-.

**Comment by acenturyandabit** [comment]
To load your own type declaration file:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/12168#issuecomment-255897170
It would be nice for someone to make a PR to DefinitelyTyped once #38 gets merged. I might do it idk

**Comment by nufaylr** [comment]
@acenturyandabit thank you. We have not updated for a while. @isaias-freitas-voith Yes we can pass those SVG XML props would be nice. I will check the PR and comeback.

**Comment by nufaylr** [comment]
@acenturyandabit regarding types I have started next version 2.xx with types. 

And for the current version 1.xx can use type definition. It's written by Rafał Witczak @rafw87.

https://www.npmjs.com/package/@types/react-svg-pan-zoom-loader

**Comment by nufaylr** [comment]
Merged and update into 1.4.2. thank you
